make.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
And remove these new make files? Unless they are necessary for this pull request.

Reply:
Actually, with Geant4 radioative decay doesn't properly compile with the "standard compilation" way. It's quite important to use CMake. For this purpose make.sh and make_sukap.sh are here to show how to do it. I can remove them if you prefer.